build.xml

```diff
@@ -2205,6 +2238,8 @@
 
   <!-- Generate IDEA project description files -->
   <target name="generate-idea-files" depends="init,maven-ant-tasks-init,resolver-dist-lib,gen-cql3-grammar,generate-jflex-java,createVersionPropFile" description="Generate IDEA files">
+    <delete dir=".idea"/>
```

Comment:
This is already in 5.0 after discussion on the mailing list - https://lists.apache.org/thread/o2fdkyv2skvf9ngy9jhpnhvo92qvr17m
Do you want to change it in datastax/cassandra all branches? It seems there was consensus. 
By the way what config files for testing you have that may be useful and we can actually add for everyone? maybe?

Reply:
If someone runs `ant generate-idea-files` that says to me they want to start from freshly generated files?  Otherwise why are you running it?

Reply:
We run it when we switch between JDK versions for example

Reply:
For example, I run `generate-idea-files` to update the libraries when updating or switching branch - I don't want to lose all the run configurations I had

Reply:
actually this is the scenario which always annoyed me: I've added/removed a dependency (could by branch update or explicitly) - I need to do clean and generate idea files again. I'm losing everything, all the setups - for example I had test configurations prepared to run with compression or something else, that is gone. Now, it will be gone even without explicitly calling realclean. 

`generate-idea-files` will be even worse now - it still does not refresh the libraries directory (only appends new libraries) but it removes the custom configurations

I understand the motivation, but I'm sceptical about side-effects. Can we not include this change here?

Reply:
> Now, it will be gone even without explicitly calling realclean.

On my end I almost never can switch branches properly without using `realclean`.

> generate-idea-files will be even worse now - it still does not refresh the libraries directory (only appends new libraries) but it removes the custom configurations

I honestly never hit that problem. I always do `ant realclean ant jar ant generate-idea-files` when switching branches. Only the case of config makes sense to me and you made me wonder whether we want to add compression config somehow.

> I understand the motivation, but I'm sceptical about side-effects. Can we not include this change here?


I can do it for `main`, but it is already accepted in Apache and it is in `main-5.0`. Do you suggest to remove it in main-5.0 too? So far I heard positive feedback from others but I want to make your life easy too. Maybe we open follow up issue to think how we can improve explicitly that target? And for now we just remove the deletion part here and leave things as they are in `main-5.0`? Let me know, thanks